code/style.css

```diff
@@ -1,4 +1,349 @@
 html {
-  background: #0025ff;
-  color: #fff;
+    box-sizing: border-box;
 }
+
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+body {
+    background: linear-gradient(45deg, #fb3222,#b40000,#890000,	#540000, #5e1712, #240000);
+    color: #fff;
+    font-family: "Titillium Web", "Helvetica Neue", "Helvetica", Arial, sans-serif;
+    margin: 0;
+    background-size: 400%;    animation: bg-animation 3s infinite alternate;
+}
+
+@keyframes bg-animation {
+    0% {
+        background-position: left;
+    }
+    100%{
+        background-position: right;
+    }
+}
+
+.container {
+    width: 420px;
+    height: 100%;
+    margin: 25px auto;
+}
+
+
+/*** Header ***/
+
+.header {
+    background-color: #2a2c36;
+    height: 70px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 auto;
+}
+
+.logo-image {
+    width: 80px;
+    margin-left: 16px;
+}
+
+.nav-bar {
+    display: none;
+}
+
+.nav-hamburger-container{
+    display:flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.navigation-element {
+    color: #fff;
+    text-decoration: none;
+    text-transform: uppercase;
+    margin-left: 16px;
+}
+
+.logo {
+    font: 35px bold;
+    color: white;
+    padding: 0 40px;
+    white-space: nowrap;
+    display:flex;
+    position: relative;
+}
+
+.logo::before{
+    content: "Winter NEWS";
+    color: #fb3222;
+    width:0;
+    position: absolute;
+    overflow: hidden;
+    transition: width 0.9s;
+}
+
+.logo:hover::before{
+    width:100%;
+}
+
+.hamburger-menu {
+    display: flex;
+    width: 30px;
+    height: 30px;
+    flex-direction: column;
+    justify-content: space-between;
+    margin: 16px;
+}
+
+.hamburger-menu div {
+    width: 30px;
+    height: 5px;
+    background: #fff;   
+}
+
+.hamburger-menu-img-container {
+    display: flex;
+    flex-direction: row;
+}
+
+
+/*** Big news ***/
+
+.big-news-container{
```

Comment:
space before the { 
(This is really picky comments but get into habit as soon as possible to make your code neat and tidy. This is something that recruiters will see when looking at your code.)